### PR TITLE
[13.0][ADD] stock_request_mrp

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
This module allows for users to be able to display manufacturing orders that have
been created as a consequence of Stock Requests.

CC @ForgeFlow

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional test. Work as expected.

**Review by AdriaGForgeFlow — APPROVED** [review]
Code review and tested on runbot :+1:

File: stock_request_mrp/hooks.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+def link_existing_mos_to_stock_request(cr):
+    env = api.Environment(cr, SUPERUSER_ID, dict())
+    stock_request_obj = env["stock.request"]
+    stock_request_order_obj = env["stock.request.order"]
+    stock_request_allocation_obj = env["stock.request.allocation"]
+    mrp_production_obj = env["mrp.production"]
+    mos_with_sr = mrp_production_obj.search([("origin", "ilike", "SR/%")])
+    logger.info("Linking %s MOs records" % len(mos_with_sr))
+    for mo in mos_with_sr:
+        stock_request = stock_request_obj.search([("name", "=", mo.origin)])
```
Comment (rousseldenis) [inline stock_request_mrp/hooks.py]: Have you tested migration with a lot of production orders and requests ? I would have done search outside loop

Comment (HviorForgeFlow) [reply stock_request_mrp/hooks.py]: I moved search outside loop as you request, it seems slightly faster :)

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1:

**Pull request merged**